charts/openebs/openebs-target-pod-failure/openebs-target-pod-failure.chartserviceversion.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: openebs-target-pod-failure
+  version: 0.1.2
+  annotations:
+    categories: OpenEBS
+    vendor: CNCF
+    repository: https://github.com/litmuschaos/chaos-charts
+    support: https://slack.openebs.io/
+spec:
+  displayName: openebs-target-failure 
+  categoryDescription: >
+     Kill the cstor target/Jiva controller pod and check if gets created again
+  keywords:
+    - Kubernetes
+    - Storage Controller
+    - OpenEBS
+  Platforms:
+    - Gke
+    - Openshift
+  maturity: alpha
+  maintainers:
+    - name: shubham chaudhary
+      email: shubham.chaudhary@mayadata.io
+  minKubeVersion: 1.12.0
+  provider:
+    name: Mayadata
+  readme: "https://github.com/litmuschaos/litmus/blob/master/experiments/openebs/openebs_target_pod_failure/README.md"
```

Comment:
This link is also not working, Can you check!!

Reply:
yes, I added a note about it